test/topology_custom/test_cdc_generation_clearing.py

```diff
@@ -138,11 +140,11 @@ async def get_gen_ids() -> set[str]:
         mark = await log_file1.mark()
         gen_ids = await get_gen_ids()
         assert len(gen_ids) == 2 and first_gen_id not in gen_ids
-        await check_system_topology_and_cdc_generations_v3_consistency(manager, [host1])
+        await check_system_topology_and_cdc_generations_v3_consistency(manager, [host1, host2, host3])
```

Comment:
You could mention these changes in the commit message -- are they optional (just making the test stronger) or actually necessary

Reply:
Explained